### PR TITLE
Make global ID repository memory-aware

### PR DESCRIPTION
**Pull request by SubJunk**
I have just tested this by disabling the changes in https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1486 and scanning my whole computer, and UMS didn't run out of memory, so this works.

It's not an ideal fix - but it is much better than crashing and it will combine nicely with other fixes to come

### CODE REVIEW

File: src/main/java/net/pms/dlna/GlobalIdRepo.java [hunk 1]
```diff
@@ -19,9 +19,6 @@
 	// The number of IDs to remove at a time when memory is getting tight
 	private final int DEBOUNCE_AMOUNT = 10;
```
Comment (valib) [inline src/main/java/net/pms/dlna/GlobalIdRepo.java]: @SubJunk it could be set higher to have more memory space.

**Comment by valib** [comment]
@SubJunk sorry I read it wrong.

**Comment by SubJunk** [comment]
@valib thanks for the approval, I have to do some more testing to make sure that it doesn't break things after removing them, then I'll merge if it works well

**Comment by SubJunk** [comment]
I think this has been superceded by https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1695

**Pull request closed without merging**